pkg/kn/commands/revision.go

```diff
@@ -24,6 +24,9 @@ func NewRevisionCommand(p *KnParams) *cobra.Command {
 		Short: "Revision command group",
 	}
 	revisionCmd.PersistentFlags().StringP("namespace", "n", "default", "Namespace to use.")
+	revisionCmd.PersistentFlags().BoolVar(&p.getAllNamespaces, "all-namespaces", false,
```

Comment:
could we also share following 3 lines as well between commands?

Reply:
Do you mean that `--all-namespace` should move to root level option like `--config` and `--kubeconfig`?
I put `--all-namespace` here to align with `--namespace`(`-n`). Should I share `namespace` between commands as well?

Reply:
@nak3 probably not global, but doesnt mean we cannot reuse this code so that we dont have duplicate flag definitions.

Reply:
I'd love to see an AddNamespaceFlags fn and a GetNamespace(cmd) fn.